strawberry/field.py

```diff
@@ -18,7 +18,7 @@
     overload,
 )
 
-from cached_property import cached_property  # type: ignore
+from backports.cached_property import backports.cached_property  # type: ignore
```

Comment:
This is not the proper way to import the `cached_property` symbol. You need to do:
```suggestion
from backports.cached_property import cached_property
```

Did you test these changes?